packages/codemods/src/transforms/styled-components-to-compiled/__tests__/styled-components-to-compiled-imports.test.tsx

```diff
@@ -131,6 +131,139 @@ describe('styled-components-to-compiled imports transformer', () => {
     'it should not remove comments before transformed statement when not on top'
   );
 
+  defineInlineTest(
+    { default: transformer, parser: 'tsx' },
+    { plugins: [] },
+    `
+    import {
+      css // a single comment line
+    } from 'styled-components';
+    `,
+    `import { css/* a single comment line*/ } from '@compiled/react';`,
+    'it transforms trailing comment lines when comments are inline'
```

Comment:
I would change it into something like
'It transforms trailing line comments into block comments to have more reliable AST transformation'

Otherwise, it may be confusing for those people not aware of the [recast issue](https://github.com/benjamn/recast/issues/191#issuecomment-118104105)